spec/02-integration/20-wasm/09-filter-meta_spec.lua

```diff
@@ -37,6 +37,7 @@ describe("filter metadata [#" .. strategy .. "]", function()
 
   lazy_setup(function()
     helpers.clean_prefix()
+    helpers.clean_runtime_dir()
```

Comment:
clean_prefix  now includes cleaning the runtime dir, so this is no longer necessary (multiple occurrences of this).